src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="logo.html">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              href="apple.html"
+              class="nav_link box1 is_active"
+            >
+              apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="samsung.html"
+              class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="smartphones.html"
+              class="nav_link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="laptops&computers.html"
+              class="nav_link is_active box4"
+              data-qa="hover"
+            >
+              laptops&computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="gadgets.html"
+              class="nav_link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="tablets.html"
+              class="nav_link"
+            >
+              tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="photo.html"
+              class="nav_link"
+            >
+              photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="wideo.html"
+              class="nav_link"
+            >
+              wideo
+            </a>
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Also, it looks like the URL 'wideo.html' might be a typo. If it is meant to represent 'video', consider correcting it to 'video.html'.